atsign-foundation/atsign.dev-3.0
# Fix purgeCSS technique

**- What I did**

- purgeCSS was over-purging css causing issues when building to staging or production environments
- Replaced the technique with Hugo's [recommended approach](https://gohugo.io/hugo-pipes/postprocess/#css-purging-with-postcss)
- Added additional css data attributes to the purge whitelist

**- How I did it**

Local Changes

**- How to verify it**

Running your server in the staging environment:
```sh
npm run start -- -e staging
```

Should look the same as the development environment:
```sh
npm run start
```

**- Description for the changelog**
Fix purgeCSS technique
